content/en/graphing/guide/_index.md

```diff
@@ -13,4 +13,6 @@ disable_toc: true
 {{< whatsnext desc="Deprecated APIs:" >}}
     {{< nextlink href="graphing/guide/screenboard-api-doc" >}}Screenboard API{{< /nextlink >}}
     {{< nextlink href="graphing/guide/timeboard-api-doc" >}}Timeboard API{{< /nextlink >}}
+    {{< nextlink href="graphing/guide/dashboard-lists-api-doc" >}}Dashboard Lists API (v1){{< /nextlink >}}
```

Comment:
I think it would be less confusing to mentioned `v1` or `deprecated` in the links to minimize confusion when sharing direct links to this page. For example: `dashboard-lists-api-v1_doc`.